README.md

```diff
@@ -24,6 +24,64 @@ cmake ..
 make -j $(nproc)
 ```
 
+## Usage
+
+The daemon usage is driven primarily by configuration files whose path is passed
+to the daemon as a command line flag. The daemon accepts the following command
+line flags:
+
+| Flag | Presence     | Description                        | Supported Values                                | Examples                   |
+|------|--------------|------------------------------------|-------------------------------------------------|----------------------------|
+| `-c` | **Required** | Configuration file path            | Absolute path to the primary configuration file | `-c /etc/ztpd/config.json` |
+| `-d` | Optional     | Daemonize flag (run in background) | None (unary flag)                               | `-d`                       |
+
+### Configuration files
+
+#### Primary configuration file
+
+This file controls global daemon options, including the Wi-Fi EasyConnect device
+roles (enrollee, configurator) that are supported and activated. For each of
+these roles, a separate configuration file is specified by providing its absolute path. Each configuration file option is specified in [config.json.schema](/samples/ztpd/config/config.schema.json), and an example configuration file can be found [here](/samples/ztpd/config/config.json).
+
+#### Enrollee configuration file
+
+When the primary configuration file specifies that the Wi-Fi EasyConnect `enrollee` device role is supported, it also specifies its configuration file. This file describes enrollee options, including information about the bootstrapping key. Each configuration file option is specified in [config-enrollee.schema.json](samples/ztpd/config/config-enrollee.schema.json), and an example configuration file can be found [here](/samples/ztpd/config/config-enrollee.json).
+
+#### Configuration configuration file
```

Comment:
Should this be Configurator configuration file?

Reply:
> Should this be Configurator configuration file?

I did see and fix this at the last minute. Good catch!